### PR TITLE
blog entry about AtlasMap and Camel Designer

### PR DESCRIPTION
**Pull request by brunoNetId**
first draft to validate rendering of blog

### CODE REVIEW

File: content/blog/2022/04/camel-designer-atlasmap/index.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+# Learn more about *Camel Designer* and *AtlasMap*
+
+- Read this [*Camel Designer* blog](https://brunonetid.github.io/2020/06/28/camel-designer.html) to understand how visualisation increases productivity
+
+- A good place to get started with *Camel Designer* is its [how-to guide in GitHub](https://github.com/designer-for-camel/camel-designer/blob/master/docs/how-to.md).
+
+- Explore [*AtlasMap*'s website](https://www.atlasmap.io/) to discover all its features.
+
+- Read [this blog](https://camel.apache.org/blog/2021/11/vscode-atlasmap-release-0.1.0/) presenting the latest *VS Code AtlasMap* release
```
Comment (apupier) [inline content/blog/2022/04/camel-designer-atlasmap/index.md:78]:
```suggestion
- Read [this blog](/blog/2021/11/vscode-atlasmap-release-0.1.0/) presenting the latest *VS Code AtlasMap* release
```

File: content/blog/2022/04/camel-designer-atlasmap/index.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+---
+
+The latest release of the [VS Code Camel Designer](https://marketplace.visualstudio.com/items?itemName=brunoNetId.camel-designer) extension integrates [VS Code AtlasMap](https://marketplace.visualstudio.com/items?itemName=redhat.atlasmap-viewer): two visual tools to accelerate productivity in Camel.
+
+
+# Two visual tools for Camel integrated in VS Code
+
+[*AtlasMap*](https://www.atlasmap.io/) is a great visual data mapping tool that enables users to define data transformations that *Camel* can process at runtime.
+
+The [0.1.0 release of the *VSCode* extension for *AtlasMap*](https://camel.apache.org/blog/2021/11/vscode-atlasmap-release-0.1.0/) introduced a series of improvements that allowed a better user experience when creating and editing *AtlasMap* definitions.
```
Comment (apupier) [inline content/blog/2022/04/camel-designer-atlasmap/index.md:16]:
```suggestion
The [0.1.0 release of the *VSCode* extension for *AtlasMap*](/blog/2021/11/vscode-atlasmap-release-0.1.0/) introduced a series of improvements that allowed a better user experience when creating and editing *AtlasMap* definitions.
```

**Comment by brunoNetId** [comment]
I've updated the date of the article, and some URLs are now relative URLs.
Could you please approve to trigger a build?

**Comment by apupier** [comment]
> Could you please approve to trigger a build?

pinging some of the usual suspects @davsclaus @zregvart @oscerd

**Review by oscerd — APPROVED** [review]

**Review by davsclaus — APPROVED** [review]

**Comment by davsclaus** [comment]
@brunoNetId maybe you would like to publish this next week after the weekend, and if so update the blog date

**Comment by brunoNetId** [comment]
Thanks @davsclaus, yes, let's aim for next week... also, there are some rendering issues to be fixed.

**Comment by brunoNetId** [comment]
hi, could you please approve the workflow one more time to validate the rendering of the blog?
thanks

**Comment by brunoNetId** [comment]
hi... I've updated the blog content to resolve the issue in the workflow check.
when you can, please trigger the workflow again.

**Review by davsclaus — APPROVED** [review]

**Comment by davsclaus** [comment]
@brunoNetId lets update the date to -19 and then we can merge this

**Review by oscerd — APPROVED** [review]

**Comment by brunoNetId** [comment]
@davsclaus I believe I did already change the date to 19. On my side, I think all is good.

**Pull request merged**